endpoints/remotehost/remotehost

```diff
@@ -357,10 +355,12 @@ function launch_osruntime() {
             do_ssh $user@$host mkdir -p $container_mount/tmp
             do_ssh $user@$host mount --verbose --options bind $remote_data_dir $container_mount/tmp
             echo "Adding host directories to container mount"
-            for fs in proc dev dev/hugepages proc sys lib/modules usr/src boot; do
-                echo "Adding $fs"
+            for fs in ${chroot_rbind_mounts}; do
+                echo "Adding ${fs} to ${container_mount}"
                 do_ssh $user@$host mkdir -p $container_mount/$fs
-                do_ssh $user@$host mount --verbose --options bind /$fs $container_mount/$fs
+                do_ssh $user@$host mount --verbose --options rbind /$fs $container_mount/$fs
```

Comment:
When I switched to using `rbind` the run would work, but after the run was over the remotehost was broken.  It turns out that the cleanup `umount`(s) were failing and then when the pod was removed it would umount all of the filesystems that had been `rbind` mounted into the chroot -- so when the run was over there was no longer a `dev`, `proc`, `sysfs`, etc. filesystem mounted on the remotehost.  Debugging this was difficult given the issue with the `umount` command above never yielding any output...